scripts/coverage.sh

```diff
@@ -1,136 +1,66 @@
 #!/usr/bin/env bash
 #
-# Runs all tests and collects code coverage
+# Run tests and collect code coverage
 #
-# Warning: this process is a little slow
+# Run all:
+#   $ ./script/coverage.sh
 #
+# Run for specific packages
+#   $ ./script/coverage.sh -p solana-account-decoder
+#   $ ./script/coverage.sh -p solana-account-decoder -p solana-accounts-db [-p ...]
 
-if ! command -v grcov; then
-  echo "Error: grcov not found.  Try |cargo install grcov|"
-  exit 1
-fi
-
-if [[ ! "$(grcov --version)" =~ 0.8.[0-9] ]]; then
```

Comment:
guess this version warn is too old to keep. lol

Reply:
:trollface: